src/actions/index.js

```diff
@@ -13,9 +13,10 @@ import store from 'react-native-simple-store'
 import apiClient from 'panoptes-client/lib/api-client'
 import { PUBLICATIONS } from '../constants/publications'
 import { MOBILE_PROJECTS } from '../constants/mobile_projects'
+import { SWIPE_WORKFLOWS } from '../constants/mobile_projects'
```

Comment:
You should be able to combine lines 15-16 since they're drawing from the same location.